s2_jv_electric_boogaloo/gameloop.h

```diff
@@ -37,6 +47,11 @@ class Gameloop
 	// Directpr values
 	long long directorRandom = 1;
 	int directorFunds = 0;
+
+public slots:
+
+signals:
+	void gridUpdate(char grid[5][10]);
```

Comment:
Moins fan de magic numbers 5 et 10, est-ce qu'on peut remplacer ceci par les contantes de dimensions? Mettons qu'on les change, je veux que tout soit update en même temps

Reply:
fuck forgot to change that